pkg/retry/retry.go

```diff
@@ -5,9 +5,199 @@
 
 package retry
 
-import "time"
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/dapr/dapr/pkg/logger"
+	"github.com/pkg/errors"
+)
 
 const (
 	DefaultLinearBackoffInterval = time.Second
 	DefaultLinearRetryCount      = 3
 )
+
+//RetryStrategy is the type that represents Retry Policy settings
+type RetryStrategy string
```

Comment:
Ha, so you have the type. Please, use it instead of string in other places too.

Reply:
Fixed